Library/Homebrew/exceptions.rb

```diff
@@ -214,7 +214,7 @@ def dump
     puts
     unless RUBY_VERSION < "1.8.7" || issues.empty?
       puts "These open issues may also help:"
-      puts issues.map{ |i| "#{i['title']} (#{i['html_url']})" }.join("\n")
+      puts issues.map{ |i| "#{i['title']}: #{i['html_url']}" }.join("\n")
```

Comment:
Final thing: let's drop the `:` and we'll just merge without the brackets, thanks.